AsyncDisplayKit/ASCellNode.mm

```diff
@@ -125,6 +125,8 @@ - (void)__setNeedsLayout
 {
   CGSize oldSize = self.calculatedSize;
   [super __setNeedsLayout];
+  
+  ASDN::MutexLocker l(_propertyLock);
```

Comment:
I'm not sure why we need to lock here. `calculatedSize` setter already acquires the lock internally.